code/src/layouts/header/components/header-toolbar/HeaderToolbar.tsx

```diff
@@ -9,14 +11,28 @@ import AppIconButton from "@/components/app-icon-button/AppIconButton";
 import AppLogo from "@/components/app-logo/AppLogo";
 import AppLink from "@/components/app-link/AppLink";
 import AppTypography from "@/components/app-typography/AppTypography";
+import AppInputWithIcon from "@/components/app-input-with-icon/AppInputWithIcon";
 
 import { useModalContext } from "@/context/ModalContext";
 
 import "@/layouts/header/components/header-toolbar/HeaderToolbar.scss";
 
 const HeaderToolbar = () => {
+  const [searchValue, setSearchValue] = useState("");
   const { openModal } = useModalContext();
 
+  const handleSearchChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
should be `ChangeEvent` instead `React.ChangeEvent` just import it

Reply:
Fixed